### PR TITLE
initial version

### PR DESCRIPTION
**Pull request by FabricioFFC**
What includes:

* find path in locales files, where path can be a string ("home") or a string as a path ("home.title")
* returns error when path is not found in any locale file
* accepts localesPath and translateFunctionName as params

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
+const pathLocalizator = require('./src/path-localizator');
```
Comment (darthaud) [inline index.js]: I think is better use pathLocator instead of pathLocalizator.

File: src/path-localizator.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+      filesWherePathIsNotFound.push(filename);
+    }
+  });
+
+  return filesWherePathIsNotFound;
+};
+
+module.exports = function pathLocalizator(path, locales) {
+  const translations = getTranslations(locales);
```
Comment (darthaud) [inline src/path-localizator.js]: Maybe you can execute getTranslations method on create to improve performance

File: index.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+const runRule = (path, locales, context, node) => {
+  const result = pathLocalizator(path, locales);
```
Comment (darthaud) [inline index.js]: Maybe you can stop execution of runRule when path is empty.

**Comment by FabricioFFC** [comment]
@darthaud thanks! I solved the issues that you reported: https://github.com/natura-cosmeticos/eslint-plugin-i18n-checker/pull/1/commits/64a4464eef0fedc01f70836f03fe739e20aa956b

**Comment by darthaud** [comment]
Looks good to me.

**Review by darthaud — APPROVED** [review]
Looks good to me

**Pull request merged**